routes/api.php

```diff
@@ -17,6 +17,7 @@
     Route::put('/users/me', 'Api\UserController@update');
     Route::get('/users/me', 'Api\UserController@show');
     Route::post('/orders', 'Api\OrderController@store');
+    Route::get('/orders/{id}/getItems', 'Api\OrderController@show');
```

Comment:
I don't understand `getItems`?

Reply:
This is get order item of another order

Reply:
https://github.com/AT-PHPIntership/food-market-order/pull/214/files

Reply:
How do I should? Thank you

Reply:
Just use `/orders/{id}/items`